### ISSUE TITLE
[Feature] Make easyclick/easyclick_all work on all monitors during <taskview>

### ISSUE THREAD

**Issue by Salted-Bread** [pit-ray/win-vind#206]
When using <taskview> you can use EasyClick to select the window you want, however it only seems to show the hints on the main monitor. It would be nice if it were to show the hints on all monitors during <taskview>. I tried using <easyclick_all>, but it didn't seem to work during task view. If that's not possible having an option to bind a custom key instead of using enter for selecting a window during <switch_window> would be nice.

**Comment by pit-ray** [comment]
I have added a macro capability to `<switch_window>` since it seems to close when `<easyclick_all>` is executed on `<taskview>`. 

You can use the hint-selection by configuring a mapping like `map <c-w>s <switch_window><easyclick><click_left>` on the following build.
[win-vind_build_138_206_64bit_portable.zip](https://github.com/pit-ray/win-vind/files/12905368/win-vind_build_138_206_64bit_portable.zip)

If there are any bugs, please report it.

Thanks.

**Comment by Salted-Bread** [comment]
It's working pretty well. I used it throughout yesterday and it made switching windows very comfortable, thanks!

The only thing I noticed is that it will sometimes show two letters over a window and pressing either of them focuses it. I think it's always putting a hint in the center, so it's probably showing that center hint next to the actual window hint. Not really a big issue in my opinion.

I would attach a screenshot, but I can't figure out how to take a screenshot with the hints during the window switching.
